src/components/Menu/Menu.mdx

```diff
@@ -111,7 +111,8 @@ const exampleListItemsData = [
 `} language="tsx" />
 
 All `ActionItem`s require an `onClick` function, but the `media` prop is optional. Media will display
-to the left of the menu text, and will render either an `Icon` or `Image`. The `media` prop requires a type (icon or image)
+to the left of the menu text, and will render either an {<Link href="https://nypl-design-system.vercel.app/?path=/docs/components-media-icons-icon--docs"target="">Icon</Link>}
+or {<Link href="https://nypl-design-system.vercel.app/?path=/docs/components-media-icons-image--docs" target="">Image</Link>}. The `media` prop requires a type (icon or image)
```

Comment:
Use the production link: https://nypl.github.io/nypl-design-system/reservoir/v2/